Xcode-config/Shared.xcconfig

```diff
@@ -0,0 +1,38 @@
+#include "DEVELOPMENT_TEAM.xcconfig"
+
+// Create the file DEVELOPMENT_TEAM.xcconfig
+// in the "Xcode-config" directory within the project directory
+// with the following build setting:
+// DEVELOPMENT_TEAM = [Your TeamID]
+
+// Hint: recent Xcode versions appear to automatically create an empty file 
+// for you on the first build. This build will fail, or course, 
+// because code-signing can’t work without the DEVELOPMENT_TEAM set. 
+// Just fill it in and everything should work. 
+
+// You can find your team ID by logging into your Apple Developer account
+// and going to
+// https://developer.apple.com/account/#/membership
+// It should be listed under “Team ID”.
```

Comment:
For my case this actually didn't work because the team my account is on is a company team, so using that team ID for a personal project isn't appropriate. I found my personal team ID by manually selecting my personal team in Xcode, copying the line it added to the project file, and pasting it into DEVELOPMENT_TEAM.config.